scripts/pipeline/create_patch_pairaligner_conf.pl

```diff
@@ -320,11 +324,7 @@ =head2 GENERAL CONFIGURATION
     print " 'chunk_size'            => " . $dna_collection->{$ref_mammal}{'chunk_size'} . ",\n";
     print " 'overlap'               => " . $dna_collection->{$ref_mammal}{'overlap'} . ",\n";
     print " 'include_non_reference' => " . $dna_collection->{$ref_mammal}{'include_non_reference'} . ",\n";  
-    if ($dna_collection->{$ref_mammal}{'masking_options_file'}) {
-        print " 'masking_options_file'  => " . $dna_collection->{$ref_mammal}{'masking_options_file'} . "\n";
-    } else {
-        print " 'masking_options'       => " . $dna_collection->{$ref_exception}{'masking_options'} . "\n";
-    }
+    print " 'masking'               => " . $dna_collection->{$ref_exception}{'masking'} . "\n";
```

Comment:
I don't understand this line for a few reasons:
1. why are we using the exception masking, despite all the other values in this section coming from the mammal settings? I can see that's what we did historically, but any idea why?
2. the default exception and mammal settings are the exact same anyway, so why are we even separating them?

Reply:
I can't find any explanation in the git history. I would consider this to be a bug and that we were lucky the masking parameters were anyway the same.
I would leave the two exception mechanisms there for the time being, as it gives in theory more configuration options, but I agree this should change if/when we refactor the configuration of LastZ pipeline